lib/travis/api/serialize/v2/http/user.rb

```diff
@@ -37,13 +37,23 @@ def user_data
                   'synced_at' => format_date(user.synced_at),
                   'correct_scopes' => Github::Oauth.correct_scopes?(user),
                   'created_at' => format_date(user.created_at),
+                  'first_logged_in_at' => format_date(user.first_logged_in_at),
+                  'secure_user_hash' => secure_user_hash,
                   'channels' => channels
                 }
               end
 
               def channels
                 ["private-user-#{user.id}"]
               end
+
+              def secure_user_hash
+                OpenSSL::HMAC.hexdigest(
```

Comment:
should this section maybe be:
```
if ENV['USER_HASH_SECRET_KEY']
  OpenSSL::HMAC.hexdigest(
    'sha256',
    ENV['USER_HASH_SECRET_KEY'] || 'USER_HASH_SECRET_KEY',
    "#{user.id}"
  )
else
null
end
```
thoughts?

Reply:
I was thinking of excluding the property altogether if `USER_HASH_SECRET_KEY` is absent, what do you think of that?

Reply:
I think we are safe with the code, if I understand correctly it will either use `ENV['USER_HASH_SECRET_KEY']` or the string `'USER_HASH_SECRET_KEY'` if there's no ENV var